internal/cache/db.go

```diff
@@ -359,7 +359,8 @@ func getActiveUsers(procDir string) (activeUsers map[string]struct{}, err error)
 
 		u, err := user.LookupId(strconv.Itoa(int(stats.Uid)))
 		if err != nil {
-			return nil, err
+			slog.Warn(fmt.Sprintf("Could not map active user ID to an actual user: %v", err))
+			continue
```

Comment:
Can you add a comment mentioning that this is to avoid "ghost" UIDs (i.e. won't affect any users related to our cache)?